### ISSUE TITLE
Home Page - Missing ALT Text for the "Three Workers" Decorative Image

### ISSUE THREAD

**Issue by jfc3-dol** [presidential-innovation-fellows/worker-dol-gov#9]
While testing using the DHS Trusted Tester process, I found the decorative image of the three workers is missing ALT attribute altogether. You either need to add **alt=""** or make the image CSS background image if it is there for decoration only.

If adding ALT text is should be the following:

- Illustration of three workers working - worker digging with shovel, worker cooking, and worker working on computer

Section 508 - 22(a)
WCAG 2.0 - 2.2 Web: Images

**Comment by jfc3-dol** [comment]
Here is a screenshot of the three workers missing ALT text.

![worker dot gov - three workers iamge missing alt text](https://cloud.githubusercontent.com/assets/10423487/19775421/4de783ca-9c3e-11e6-80fe-bf578b1aaed4.png)

**Comment by adamsfallen** [comment]
Updated with alt text "Worker Icons"

**Comment by jfc3-dol** [comment]
Yes, that works for me. Closing issue.
